bors.toml

```diff
@@ -1,3 +1,3 @@
-status = ["buildbot/yk-buildbot-build-script-hw"]
-timeout_sec = 300 # 10 minutes
```

Comment:
Curious that builds were working. I'd have thought building LLVM took longer than 10 minutes...

Reply:
That's 5 minutes! I think we can assume it was ignored somehow...

Reply:
Good question. It is excessive right now, but if we start doing `try_except` it might not be quite so bonkers. That said, probably 2-3 hours would be more sensible?

Reply:
It's rather vague...

Reply:
It's definitely worked in the past...